packages/pigeon/lib/objc_generator.dart

```diff
@@ -843,7 +843,7 @@ String _className(String? prefix, String className) {
 String _callbackForType(TypeDeclaration type, _ObjcPtr objcType) {
   return type.isVoid
       ? 'void (^)(NSError *_Nullable)'
-      : 'void (^)(${objcType.ptr.trim()}_Nullable, NSError *_Nullable)';
+      : 'void (^)(${objcType.ptr}_Nullable, NSError *_Nullable)';
```

Comment:
I thought this was intentional, but I forget the details of why. What is the output change you are trying to make here?

Reply:
This was causing a bug with `id`s (which don't have trailing `*`) connecting to the `_Nullable` after. After making this change there was no difference in any of our gen files except where this bug was occuring.